openmetadata-service/src/main/java/org/openmetadata/csv/EntityCsv.java

```diff
@@ -721,6 +726,9 @@ protected void createEntity(CSVPrinter resultsPrinter, CSVRecord csvRecord, T en
         repository.prepareInternal(entity, false);
         PutResponse<T> response = repository.createOrUpdate(null, entity);
         responseStatus = response.getStatus();
+        AsyncService.getInstance()
```

Comment:
@mohityadav766 can we just collect all the responses and hand over at the end instead of invoking the async serivce everytime for each row